internal/pkg/probe/file_test.go

```diff
@@ -0,0 +1,39 @@
+package probe
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// I think we can use this test to be FileExist unit test function as well.
+func TestWriteFile(t *testing.T) {
+	var fileName string = "testfile"
+	var expected bool = true
+	WriteFile(fileName)
+
+	// check is file exist
+	result, err := FileExists(fileName)
+	require.NoError(t, err)
+
+	// delete file afterwards
+	DeleteFile(fileName)
+
+	assert.Equal(t, expected, result)
```

Comment:
same comment here

Reply:
I made the changes, please kindly check. thank you